### ISSUE TITLE
Proxy changes not working in network settings in 1.6.0

### ISSUE THREAD

**Issue by drekka** [ProxymanApp/Proxyman#275]
### Proxyman version? (Ex. Proxyman 1.4.3)

1.6.0

### macOS Version? (Ex. mac 10.14)

10.14.6

### Steps to reproduce 

1. Start Proxyman.
2. Start proxy interception.
3. Check network settings
4. Stop Proxyman
5. Check network settings.

### Expected behavior

Under Proxyman 1.5.1 I would see my 'Automatic Proxy Configuration' disabled and 'Web proxy' and 'Secure web proxy' enabled and pointed at Proxyman. When stopping Proxyman I would see the settings reverted. I could even see it occur right in front of me if I had the dialog open.

When I installed 1.6.0 I also un-installed and re-installed the NSProxy helper tool. But when I check the settings, it looks like Proxyman is setting itself up correctly the first time, but after that it fails to change the settings and I have to change them manually. 

I also noticed that it seems to clear the url of my pac file. That may be technically correct, but it's kind of annoying due to my previous bug report on settings not being restored. :-)

**Comment by drekka** [comment]
Just been playing around. It looks like the update to add the 'Override MacOS proxy' has disabled the updating of the proxy settings when I use the start/stop button. Is this the intended behaviour?

Also using the menu item instead of the start/stop button is not restoring the settings at all so it looks like there's still a bug in there somewhere.

**Comment by NghiaTranUIT** [comment]
> Just been playing around. It looks like the update to add the 'Override MacOS proxy' has disabled the updating of the proxy settings when I use the start/stop button. Is this the intended behaviour?
Yes, it's our intention in [Proxyman 1.6.0](https://github.com/ProxymanApp/Proxyman/releases/tag/1.6.0). We separately the Overridden Proxy and the Start/Stop since it make sense than previous version. Same behavior with Charles Proxy.

For instance, some user would:
1. Only capture traffic in remote devices, and ignore all traffic in macOS. In previous version, there is no way to turn OFF the Override Proxy without stopping the app (Start/Stop button will stop the app and revert Proxy too). Thus, they had to revert the Proxy manually in Network.prefPanel. 
2. Stop Proxyman cause the mobile couldn't access the Internet since the port 9090 is closed.

Therefore, win Proxyman 1.6.0 we update
1. they can easily turn off the Proxy on mac by Option+Cmd+O and keep recording traffic in their iPhone 😄 
2. Mobile device is still working since the port 9090 is not closed.

-------------------------------
> Also using the menu item instead of the start/stop button is not restoring the settings at all so it looks like there's still a bug in there somewhere.
Yup, it's related on #272. I'm investing it now 👍

**Comment by NghiaTranUIT** [comment]
As I mention before, it's our intention to separate the Override macOS Proxy and the Start/Stop app to solve some user-case which it make sense.

If you would a hot-fix build, let check https://github.com/ProxymanApp/Proxyman/issues/272#issuecomment-529538637
Thank you in advance 😄

**Comment by shirshak55** [comment]
@NghiaTranUIT yea agree with it :) most of the time i use proxy for andriod apps only .

**Comment by NghiaTranUIT** [comment]
Just friendly reminder @shirshak55 and @drekka that I've released [Proxyman 1.6.1](https://github.com/ProxymanApp/Proxyman/releases/tag/1.6.1) 🎉. 

Let check it out 😄 

If you're happy with it, please close this ticket @drekka 🌮
